articles/react-native-agora-livestreaming-app/index.md

```diff
@@ -0,0 +1,681 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/react-native-agora-livestreaming-app/
+title: React Native Livestream Application using Agora
+description: This tutorial will give readers a detailed guide on how they can build an app to create or join a live stream using Agora.
+author: mohan-raj
+date: 2020-11-19T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/react-native-agora-livestreaming-app/hero.jpg
+    alt: Python example image PyPi
+---
+Want to learn how to build a cool and simple Livestreaming app using React Native? In this tutorial, we will do so using [Agora](https://www.agora.io/en/about-us/), which is a video, voice, and live interactive streaming platform. It includes embedded voice and video chat, real-time recording, interactive live streaming, and real-time messaging.
+<!--more-->
+Keep reading to find out.
```

Comment:
Unnecessary

Reply:
Do you mean just the "Keep reading to find out"?

Reply:
removed "Keep reading to find out".